### ISSUE TITLE
Not all scenes are displayed

### ISSUE THREAD

**Issue by GFR3** [pgerke/homebridge-freeathome-local-api#99]
**Describe The Bug:**
Not all scenes show up in homebridge

**To Reproduce:**
Scene ID FFFF4803….

**Expected behavior:**
All scenes should be display

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**: 1.11.4
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by pgerke** [comment]
At the moment only "regular" scenes with `0x4800` function IDs are supported. In the meantime Busch+Jaeger added the folloginw new special scenes:

~~~
0x4801	FID_SPECIAL_SCENE_PANIC
0x4802	FID_SPECIAL_SCENE_ALL_OFF
0x4803	FID_SPECIAL_SCENE_ALL_BLINDS_UP
0x4804	FID_SPECIAL_SCENE_ALL_BLINDS_DOWN
~~~

I'll consider supporting those as an enhancement.

**Comment by GFR3** [comment]
Thanks for the quick response

That would be great if you could add those.

**Comment by pgerke** [comment]
I have enabled the additional scenes and published a preview version [v1.12.0-pre.0](https://www.npmjs.com/package/homebridge-freeathome-local-api/v/1.12.0-pre.0). @GFR3, please feel free to check it out, any feedback would be much appreciated!

**Comment by GFR3** [comment]
Thank you! I will give it a try in the upcoming week and let you know the result.

Grüße aus der Oberpfalz 😉

**Comment by GFR3** [comment]
Gave it a try, but scenes still do not show up.

**Comment by pgerke** [comment]
Please try again, there was a change that I forgot to include in the initial commit. It's working for me with [v1.12.0-pre.1](https://www.npmjs.com/package/homebridge-freeathome-local-api/v/1.12.0-pre.1).

**Comment by GFR3** [comment]
Works! 👍

**Comment by pgerke** [comment]
Happy to hear that! The change is now officially released as [v1.12.0](https://www.npmjs.com/package/homebridge-freeathome-local-api/v/1.12.0).
